### DISCUSSION TITLE
malware code

### DISCUSSION THREAD

**Discussion by hoon75** [mkaring/ConfuserEx#459] [Q&A]
My project is Recognized as malware code and deleted

**Comment by KvanTTT** [comment]
Duplicate:

https://github.com/mkaring/ConfuserEx/issues/64
https://github.com/mkaring/ConfuserEx/issues/148
https://github.com/mkaring/ConfuserEx/issues/57
https://github.com/mkaring/ConfuserEx/issues/113

**Reply by mkaring** [reply]
Issue #64 contains the most detailed answer for that question. :wink:
